### PR TITLE
Updated frontend/backend tests

### PR DESCRIPTION
**Pull request by karthikmsqa**
# Description

This PR includes the following proposed change(s):

- Updated tests on CSO account endpoint
- Updated frontend tests

## Type of change

- [x] New tests (non-breaking change which adds tests)

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

On localhost in demo mode.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by akroon3r** [comment]
-50 DKP for PR during sprint planning

File: tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/createcsoaccountpayload/CsoPayload.java [hunk 1]
```diff
@@ -2,20 +2,32 @@
...
     private String lastName;
     private String middleName;
     private String email;
     Accounts[] accounts;
 
-    public CsoPayload(String firstName, String lastName, String middleName, String email, Accounts[] accounts) {
+    public CsoPayload(String universalId, String firstName, String lastName,
+                      String middleName, String email, Accounts[] accounts) {
+
+        this.universalId = universalId;
```
Comment (akroon3r) [inline tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/createcsoaccountpayload/CsoPayload.java:15]: maybe change this to uid? the 'l' with the "I" back to back is panic inducing. any thoughts on this anyone else?

Comment (karthikmsqa) [reply tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/createcsoaccountpayload/CsoPayload.java:15]: This is a pojo class to create the payload so the fields need to match the json object. Thanks.

Comment (sdevalapurkar-bcgov) [reply tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/createcsoaccountpayload/CsoPayload.java:15]: we use `universalId` on fe as well so im ok with it :)

File: tests/src/test/java/stepDefinitions/frontendstepdefinitions/EFileSubmissionTest.java [hunk 2]
```diff
@@ -192,7 +199,7 @@ public void errorMessageIsDisplayed() {
         landingPage.clickGenerateUrlButton();
         log.info("Generate Url button in eFiling frontend page is clicked");
 
-        String expMsg = "An error occurred while generating the URL. Please try again.";
+        String expMsg = "An error occurred while eFiling your package. Please make sure you upload at least one file and try again.";
```
Comment (sdevalapurkar-bcgov) [inline tests/src/test/java/stepDefinitions/frontendstepdefinitions/EFileSubmissionTest.java:202]: this has changed in master now :)

**Pull request merged**